dexs/ashswap/index.ts

```diff
@@ -0,0 +1,33 @@
+import { request } from "graphql-request";
+import { SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import { getUniqStartOfTodayTimestamp } from "../../helpers/getUniSubgraphVolume";
+
+const API_URL = 'https://api-v2.ashswap.io/graphql';
+
+const VolumeQuery = `
+{
+  defillama {
+    totalVolumeUSD24h
+  }
+}
+`
+
+const fetch = async (timestamp: number) => {
+  const results = await request(API_URL, VolumeQuery);
+  return {
+    dailyVolume: results.defillama.totalVolumeUSD24h,
+    timestamp: getUniqStartOfTodayTimestamp(new Date(timestamp * 1000)),
+  };
+}
+
+const adapter: SimpleAdapter = {
+  adapter: {
+    [CHAIN.ELROND]: {
+      fetch: fetch,
+      start: async () => 1676606400
```

Comment:
could you set start is `1676419200` for test yesterday ?

Reply:
actually ashswap start lauching at Feb 17 16h UTC, so the first volume 24h from Feb 18 16h UTC
in this case, which start time I should use

Reply:
I tested yesterday: Subgraph return volume is 0

Reply:
Sorry sir, on Feb 18, 16 UTC user are allowed to trade, so volume starts from this time, first 24h volume since Feb 19, 16 UTC.

Reply:
ok I will check again tomorrow